content/_code-samples/freeze/py/freeze_token.py

```diff
@@ -0,0 +1,40 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import IssuedCurrencyAmount, TrustSet
+from xrpl.models.transactions import TrustSetFlag
+from xrpl.transaction import (safe_sign_and_autofill_transaction,
+                              send_reliable_submission)
+from xrpl.wallet import generate_faucet_wallet
+
+client = JsonRpcClient("https://s.altnet.rippletest.net:51234") # connect to testnet
+
+# token name
+token = "FOO"
+
+# amount a trustline can handle, for this transaction it is set to 0
+value = "0"
+
+# address to freeze trustline
+target_addr = generate_faucet_wallet(client=client).classic_address
+
+# generate wallet
+sender_wallet = generate_faucet_wallet(client=client)
+
+# build trustline freeze transaction
+trustset = TrustSet(account=sender_wallet.classic_address, limit_amount=IssuedCurrencyAmount(
+    currency= token,
+    issuer=target_addr,
+    value = value),
+    flags=TrustSetFlag.TF_SET_FREEZE) # flag to freeze the trust line. check all trustset flags here https://xrpl.org/trustset.html#trustset-flags
```

Comment:
Please move this comment to it's own line - as of now it goes past the edge of the screen which makes it hard to read.